### PR TITLE
fix(storefront): BCTHEME-325 Apple pay button displaying needs to be fixed

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
#### What?

Inline block display for preview cart button styling overrode conditional display of Apple Pay button and resulted in an empty button when Apple Pay was not available. Applied conditional rules used from cart styling to preview cart.

#### Tickets / Documentation

[Jira](https://jira.bigcommerce.com/browse/BCTHEME-325)

#### Screenshots (if appropriate)

Apple Pay disabled 
Screenshot 2021-04-26 at 13 29 19

Apple pay enabled
Screenshot 2021-04-26 at 13 29 11

### CODE REVIEW

**Comment by BC-tymurbiedukhin** [comment]
@bc-azvierieva Please verify this fix with Apple Pay button enabled

**Comment by BC-tymurbiedukhin** [comment]
@bc-azvierieva PR is updated, I believe you could proceed with verifying of this issue

**Comment by bc-azvierieva** [comment]
@BC-tymurbiedukhin checked with latest commit, Apple Pay button is now visible on Added to cart page 

image

Though Apple pay button is still missing in the Cart preview. 
Actual:

image

Expected:

image

The same with the Cart page. Actual:

image

Expected:

image

**Comment by BC-tymurbiedukhin** [comment]
@bc-azvierieva Apple pay button has been added to cart page and cart preview dropdown

**Comment by bc-azvierieva** [comment]
@BC-tymurbiedukhin button displaying is now looks good, though nothing happens after clicking on Apple pay button. On Prod store it opens payment window, on local store it doesn't. 

Actual: https://www.screencast.com/t/J1L7JG4Yxz7d
Expected: https://www.screencast.com/t/yXscpmOypAA3

**Comment by BC-tymurbiedukhin** [comment]
@bc-azvierieva Apple Pay button handling is behind theme project, I believe it could be the reason why button was hidden for cart preview and cart pages before.

**Review by bc-azvierieva — APPROVED** [review]

**Comment by BC-tymurbiedukhin** [comment]
@yurytut1993 @bc-alexsaiannyi Could you please have a look from your side? Thanks

**Review by bc-alexsaiannyi — APPROVED** [review]

File: assets/js/theme/common/product-details.js [hunk 1]
```diff
@@ -376,6 +376,10 @@ export default class ProductDetails extends ProductDetailsBase {
             if (this.previewModal) {
                 this.previewModal.open();
 
+                if (window.ApplePaySession) {
+                    this.previewModal.getJqueryElement().addClass('apple-pay-supported');
+                }
```
Comment (yurytut1993) [inline assets/js/theme/common/product-details.js:381]: Can we get access to the element like this.previewModal.$modal&?

Comment (BC-tymurbiedukhin) [reply assets/js/theme/common/product-details.js]: Updated

**Review by yurytut1993 — APPROVED** [review]

**Pull request merged**